### ISSUE TITLE
Timing Connection Report Map

### ISSUE THREAD

**Issue by ODOT-RPTD-mb** [ODOT-PTS/TNAST_MAVEN#111]
http://tna.trilliumtransit.com:8090/TNAtoolAPI-Webapp/TimingConnection.html?&dbindex=11
Map not displaying.

**Comment by irees** [comment]
Possible causes:
1. Missing Google Maps API keys
2. 'Show Map' button may attempt to render before some request callbacks are complete
3. Map will not display unless window is at least a certain width
4. Strange behavior with buttons twitching between clickable/not clickable, possibly callbacks/rendering related.

Spending more time in the debugger. Fortunately under some circumstances and waiting patiently between each click, the map will render. This will make it a bit easier to track down the error, and the issue is entirely front-end related.

**Comment by irees** [comment]
This took a bit of sleuthing.

The timing connection report main frame handled the queries and table display, and then shared the javascript state with an iframe that contained the map. This iframe was reloaded each time a trip was queried; this was the source of the problems, and became very dependent on the order of events, timing, etc. I ended up moving the map from the iframe into the main timing connection report html and javascript, and improving the handling of the map initialization and updates after each trip query.

I also added a feature to zoom the map to all the displayed trips.
